src/main/java/com/novoda/notils/caster/Fragments.java

```diff
@@ -1,31 +0,0 @@
-package com.novoda.notils.caster;
-
-public final class Fragments {
-
-    private Fragments() {
-    }
-
-    /**
-     * Simpler version of {@link android.support.v4.app.FragmentManager#findFragmentById(int)}} which infers the target type.
-     */
-    @SuppressWarnings("unchecked")
-    public static <T> T findFragmentById(android.support.v4.app.FragmentManager fragmentManager, int fragmentId) {
-        return (T) fragmentManager.findFragmentById(fragmentId);
-    }
-
-    @SuppressWarnings("unchecked")
-    public static <T> T findFragmentByTag(android.support.v4.app.FragmentManager fragmentManager, String fragmentTag) {
```

Comment:
was the only reason we brought in the support library, I think this entire file will probably go